lib/layer.js

```diff
@@ -40,7 +40,7 @@ function Layer (path, options, fn) {
   this.keys = []
   this.name = fn.name || '<anonymous>'
   this.params = undefined
-  this.path = undefined
+  this.path = path
```

Comment:
I think this property is expected to contain the matched `path` and not the possible array of paths of the